### ISSUE TITLE
Soft matching 

### ISSUE THREAD

**Issue by Zhougv** [lbcb-sci/RiNALMo#15]
In the paper there are statements about soft matching of the secondary structure prediction task, where is the code for it？

**Comment by retiro** [comment]
You can find it here:
https://github.com/lbcb-sci/RiNALMo/blob/da6572dc22617f95f7c657e79b2ea126dc7cf979/rinalmo/utils/sec_struct.py#L169

**Comment by Zhougv** [comment]
Soft matching is mentioned in the paper, but function _relax_ss is not imported in script train_sec_struct_prediction.py .

**Comment by Zhougv** [comment]
Soft matching is mentioned in the paper, but function _relax_ss is not imported in script train_sec_struct_prediction.py . Are the results of your secondary structure prediction task obtained using soft matching? In the TS0 evaluation dataset, you achieved an F1 score of 0.75, which is significantly higher than other models.

**Comment by RJPenic** [comment]
Hello, if you checked the code more carefully you would have noticed that metric methods (recall, precision, F1) utilize the `_relax_ss` method and those are imported in the training script.
